src/AutoRest.CSharp/LowLevel/AutoRest/DpgOutputLibrary.cs

```diff
@@ -19,7 +19,7 @@ internal class DpgOutputLibrary : OutputLibrary
         private readonly string _rootNamespace;
         private readonly IReadOnlyDictionary<InputEnumType, EnumType> _enums;
         private readonly IReadOnlyDictionary<InputModelType, ModelTypeProvider> _models;
-        private readonly bool _isCadlInput;
+        private readonly bool _isTspInput;
```

Comment:
Can we use  _isTypeSpecInput here?

Reply:
We will use tspOutput, tspConfig in source code, for consistent, I prefer to using isTspInput also.